modules/imgproc/test/test_intersectconvexconvex.cpp

```diff
@@ -0,0 +1,301 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
+//
+//  IMPORTANT: READ BEFORE DOWNLOADING, COPYING, INSTALLING OR USING.
```

Comment:
Please add [short license header](https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure).

You may add authors after that (however authors are already in git history):
```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
//
// @Authors
//      Thang Tran <TranKimThang279@gmail.com>
```